data/brands/amenity/fast_food.json

```
@@ -4986,6 +4993,7 @@
         "brand:wikidata": "Q1244034",
         "brand:wikipedia": "en:White Castle (restaurant)",
         "cuisine": "burger",
+        "diet:vegetarian": "yes",
```

```suggestion
        "diet:vegan": "yes",
```
White Castle has an impossible slider which is vegan.